### PR TITLE
Join tables

### PR DESCRIPTION
**Pull request by NicolasAndrehh**
# Milestone - Join Tables
This pull request includes the following changes:
- Create tables vets, specializations and visits
- Add data to the tables created previously (vets, specializations and visits)
- Add queries to answer the following questions:

>``` 
> Who was the last animal seen by William Tatcher?
>```
> image

>``` 
> How many different animals did Stephanie Mendez see?
>```
> image

>``` 
> List all vets and their specialties, including vets with no specialties.
>```
> image

>``` 
> List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.
>```
> image

>``` 
> What animal has the most visits to vets?
>```
> image

>``` 
> Who was Maisy Smith's first visit?
>```
> image

>``` 
> Details for most recent visit: animal information, vet information, and date of visit.
>```
> image

>``` 
> How many visits were with a vet that did not specialize in that animal's species?
>```
> image

>``` 
> What specialty should Maisy Smith consider getting? Look for the species she gets the most.
>```
> image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -40,4 +40,33 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
...
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+INSERT INTO public.vets(name, age, date_of_graduation)
+VALUES 
+('William Tatcher', 45, '2000/04/23'),
+('Maisy Smith', 26, '2019/01/17'),
+('Stephanie Mendez', 64, '1981/05/04'),
+('Jack Harkness', 38, '2008/06/08');
+
```
Comment (rotimiazeez) [inline data.sql:51]: Please ensure the specialization data is add to your data.sql file. 👍🏼
Screenshot_20230503_170034

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @NicolasAndrehh 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- Good file organization 👏🏾 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by NicolasAndrehh** [comment]
Thanks for the feedback @rotimiazeez, I forget to add that part 😅, I'll implement the changes now!

**Review by Munsa1 — APPROVED** [review]
Hi @NicolasAndrehh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made required changes :heavy_check_mark: 
 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**